source/blueprints/integrations/traccar_server/device_enter_geofence.yaml

```diff
@@ -0,0 +1,53 @@
+blueprint:
```

Comment:
As this is they first of many integrations to implement this, this is the time to set standards.

I looked at how this would be imported, and it seems that this would be imported to `blueprints/automation/homeassistant/device_enter_geofence.yaml`


In this repo, I placed the file as `blueprints/integrations/traccar_server/device_enter_geofence.yaml` so that kinda make sense as to whats happening.

I wonder if we should drop the `traccar_server` directory from the path, and rather prefix the files with that instead, that will still give us some form of structure, and at the same time not cause conflicts with several integrations providing a blueprint with the same name.

Reply:
Sounds good 👍